components/TravelDetails/index.js

```diff
@@ -1,23 +1,27 @@
-import { travel } from "@/data/data";
 import { styled } from "styled-components";
 import UserIcon from "../UserIcon";
 import TimePeriod from "./TimePeriod";
 
-const trip = travel[0];
-
-export default function TravelOverview() {
+export default function TravelOverview({ currentTrips, setCurrentTrips }) {
   return (
-    <FlexContainer>
-      <article>
-        {trip.name}
-        <br />
-        📍{trip.destination}
-      </article>
-      <FlexArticle>
-        <UserIcon />
-        <TimePeriod />
-      </FlexArticle>
-    </FlexContainer>
+    <>
+      {currentTrips.map((trip) => (
+        <FlexContainer key={trip.name}>
```

Comment:
If some trips would possibly have the same name, your key won't be unique. But you can also validate your trip name in the form that a trip name has already been given.